features/storage/kvstore/securestore/SecureStore.cpp

```diff
@@ -60,23 +58,21 @@ typedef struct {
 } record_metadata_t;
 
 // incremental set handle
-typedef struct {
+struct mbed::SecureStore::inc_set_handle_t {
```

Comment:
The `mbed` isn't required here - remove it to be consistent with the function definitions.